src/Instance/ItemsList/ItemsListContainer.js

```diff
@@ -42,9 +42,9 @@ const ItemsListContainer = ({
   const { totalRecords } = useHoldingItemsQuery(holding.id, { searchParams: { limit: 0 }, key: 'itemCount', tenantId });
 
   useEffect(() => {
-    if (!isEmpty(items)) {
-      setItemsToShow(items);
-    }
+    const updatedItemsList = !isEmpty(items) ? items : [];
```

Comment:
Looks like items is always an array either with length 0 or greater than 0 if there are items in holdings.
hence, do you need this isEmpty check on items? You should probably update the state when new items are available.

Reply:
@Terala-Priyanka `isEmpty` also checks for `undefined` and `null`, so if `items` is undefined nothing crushes. I need to update the state even in the case when all items have been moved to another holding and it's empty.

Reply:
@mariia-aloshyna 
Looks like API throws an array always. Do you see any case when a null is sent through API?
As long as it is array. we don't need to have this isEmpty check. directly update the state with new response.

Reply:
@Terala-Priyanka updated